lib/ansible/modules/cloud/amazon/ec2_snapshot_facts.py

```diff
@@ -185,7 +185,7 @@
 def list_ec2_snapshots(connection, module):
 
     snapshot_ids = module.params.get("snapshot_ids")
-    owner_ids = map(str, module.params.get("owner_ids"))
+    owner_ids = module.params.get("owner_ids")
```

Comment:
This drops running str on the owner_ids.  This could have an impact if the API these are used with expect strings but an int is passed in, for instance.  If the str is still needed, you can do something like:

``` python
owner_ids = [str(i) for i in module.params.get("owner_ids")]
```